ETL/ETL/_clock_system.h

```diff
@@ -29,25 +29,11 @@
 
 /* === H E A D E R S ======================================================= */
 
-#ifndef _WIN32
-# include <time.h>
-# define __sys_clock	::clock
-# define __sys_time	::time
-#else
-# ifdef __GNUG__
-#  include <time.h>
-#  define __sys_clock	::clock
-#  define __sys_time	::time
-# else
-typedef int clock_t;
-typedef int time_t;
-extern clock_t _clock();
-extern time_t _time(time_t *);
-#  define CLOCKS_PER_SEC 1000
-#  define __sys_clock	_clock
-#  define __sys_time	_time
-# endif
-#endif
+#ifndef __sys_clock
+#include <ctime>
+#define __sys_clock	::clock
```

Comment:
Could we eliminate `__sys_clock`?
Anyway, macro names starting with `__` are reserved and should not be note used. But it's used here and as headr file guards (__ETL__CLOCK_SYSTEM_H , __SYNFIG_xxxxx_H and so on).

Reply:
Sure. I can’t do it now because I have a bunch of modified files and it will be difficult for me. But you can help here if you want. I also want to come back here later, because I think that `ETL/time` can be transferred to `synfig-core`.

Reply:
Ok. :) Not for this PR anyway